src/components/swap/ConfirmSwapModal.tsx

```diff
@@ -261,6 +266,21 @@ export default function ConfirmSwapModal({
     startSwapFlow,
   ])
 
+  const l2Badge = useCallback(() => {
```

Comment:
nit: In this particular context, since this is not used in any other hook, `l2Badge` does not need `useCallback`